src/test/java/org/folio/mr/controller/MediatedRequestsControllerTest.java

```diff
@@ -28,13 +30,19 @@ void getByIdNotFoundWhenNull() {
     when(requestsService.get(any())).thenReturn(Optional.empty());
     var response = requestsController.getMediatedRequestById(any());
     verify(requestsService).get(any());
-    Assertions.assertEquals(response.getStatusCode(), HttpStatusCode.valueOf(404));
+    Assertions.assertEquals(HttpStatusCode.valueOf(404), response.getStatusCode());
```

Comment:
We can use HttpStatus enum here instead of a 404 literal (the same with 200 statuses below)

Reply:
updated